packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -1739,18 +1739,18 @@ describe('TransactionController', () => {
   });
 
   describe('initApprovals', () => {
+    const txMeta = {
+      from: ACCOUNT_MOCK,
+      hash: '1337',
+      id: 'mocked',
+      chainId: toHex(5),
+      status: TransactionStatus.unapproved,
+    };
     it('creates approvals for all unapproved transaction', async () => {
-      const txParams = {
-        from: ACCOUNT_MOCK,
-        hash: '1337',
-        id: 'mocked',
-        chainId: toHex(5),
-        status: TransactionStatus.unapproved,
-      };
       const controller = newController();
-      controller.state.transactions.push(txParams as any);
+      controller.state.transactions.push(txMeta as any);
       controller.state.transactions.push({
-        ...txParams,
+        ...txMeta,
         id: 'mocked1',
         hash: '1338',
       } as any);
```

Comment:
`as` assertion unnecessary with `txMeta` typed as `TransactionMeta`. 

```suggestion
        ...txMeta,
        id: 'mocked1',
        hash: '1338',
      });
```